src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb

```diff
@@ -2309,6 +2309,52 @@ End Class"
             locals.Free()
         End Sub
 
+        <WorkItem(1115044, "DevDiv")>
+        <Fact>
+        Public Sub CaseSensitivity()
```

Comment:
My mistake, this change is for pseudo-variables only so the test is not necessary.